docroot/themes/custom/dcp/templates/node/node--session.html.twig

```diff
@@ -58,7 +58,7 @@
     </div>
   </div>
 </div>
-  {% if content.field_attachment %}
+  {% if content.field_attachment['#items'] %}
```

Comment:
@pratik17893 this is already updated in https://github.com/drupalpune/dcp/pull/59/files#diff-4724ad5cd18afb0d819e6556dc2f826eL61 and I have merge that PR first so can you please revert this piece of code